link-all.sh

```diff
@@ -25,8 +25,10 @@ for module in ${modules}; do
   # Symlink executable scripts into place as well. This is not completely
   # according to spec (we look in the bin/ directory instead of the { "scripts"
   # } entry in package.json but it's quite a bit easier.
-  [[ -d $module/bin ]] && for script in $(find $module/bin -perm +111); do
```

Comment:
Note on this change: This format causes ./link-all.sh to return with exit code 1, because the last module doesn't have a bin directory, which fails the Docker image build.